www/badges/buildbot_badges/__init__.py

```diff
@@ -31,12 +31,12 @@ class Api:
     app = Klein()
 
     default = {  # note that these defaults are documented in configuration/www.rst
-        "left_pad": 5,
+        "left_pad": 0,
         "left_text": "Build Status",
         "left_color": "#555",
-        "right_pad": 5,
-        "border_radius": 5,
-        "style": "plastic",
```

Comment:
why change the default? please document in www.rst example configuration with required default changes.

Reply:
You're probably right, I shouldn't have changed the defaults. I've reverted those changes, while adding a documentation blurb with image.

Reply:
PS: The font style change is really required, as all templates have incorrect character widths calculated; due to the mismatch in font weights used (bold for calculation and normal weight for display.)